students/nartowska_karina/lesson_04_unit_testing/unit_tests_for_distance.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from math import sqrt
+
+
+def distance(x1, y1, x2, y2):
```

Comment:
Look at my comment to test_for_fibonacci_numbers.py. Before you will import distance function you should add a fix to it (function should return value:))